app/src/Build/BuildDocsTask.php

```diff
@@ -43,19 +45,38 @@ class BuildDocsTask extends BuildTask
         '/-theme$/',
     ];
 
+    private ?BuildDocsQueuedJob $job;
+
+    private OutputInterface $output;
+
+    public function __construct(?BuildDocsQueuedJob $job = null)
+    {
+        parent::__construct();
+        $this->job = $job;
+        $this->output = new ConsoleOutput(ConsoleOutput::VERBOSITY_VERBOSE);
```

Comment:
I've done a bit of refactoring to ensure that anything we want to log will show in the messages tab of the queued job